packages/web-frontend/cypress/config/dashboardReportUrls/default.json

```diff
@@ -1,4 +1,230 @@
 [
+  {
+    "code": "LESMIS_repetition_rate_primary_district",
```

Comment:
The urls are currently sorted by `code`. I believe it would be beneficial if we maintain this sort order, could you move them further down the file?